test/template.test.in

```diff
@@ -1,3 +1,3 @@
 [Test]
```

Comment:
By what program are these files used?
Also, we currently install these files for every platform. I suppose these are only useful for Linux?

Reply:
See #5525 for background.

> By what program are these files used?

The reference implementation is `ginsttest-runner` from <https://gitlab.gnome.org/GNOME/gnome-desktop-testing>. The specification in https://wiki.gnome.org/Initiatives/GnomeGoals/InstalledTests is fairly generic, and anyone could use it.

https://gitlab.steamos.cloud/steamrt/steam-runtime-tools/-/blob/main/debian/tests/run is a not-fully-spec-compliant subset for OSs where gnome-desktop-testing isn't available or would be annoying to install.

>  Also, we currently install these files for every platform. I suppose these are only useful for Linux?

I believe `ginsttest-runner` is meant to be portable to any Unix, for example the BSDs. It probably doesn't compile successfully on Windows, but in principle there's nothing to stop someone writing an interpreter for these files that's portable to Windows.

I think it makes sense to install the `.test` files whenever we install the test binaries themselves: at worst, they're harmless.

Reply:
Thanks! I'll leave the alone then :)

Reply:
We used to run the installed-tests in Github CI, but that was removed along with Autotools. I should resurrect that as part of the CMake build.